ga4gh/datarepo.py

```diff
@@ -23,6 +24,8 @@ def __init__(self):
         self._referenceSetIdMap = {}
         self._referenceSetNameMap = {}
         self._referenceSetIds = []
+        self._ontologyNameMap = {}
```

Comment:
Why ontologyNames? We should follow the same pattern as datasets and referenceSets unless there's a good reason not to.